.github/workflows/run-e2e.yml

```diff
@@ -38,10 +38,10 @@ jobs:
       - name: Install Playwright Browsers
         run: cd src/e2e && npx playwright install --with-deps
       - name: Run your tests
-        run: 'cd src/e2e && npm run e2e:run -- --world-parameters "{\"urls\": { \"zac\": \"${{ secrets.ZAC_URL }}\"}, \"headless\": true}"'
+        run: 'cd src/e2e && npm run e2e:run -- --world-parameters "{\"urls\": { \"zac\": \"${{ secrets.ZAC_URL }}\"}, \"headless\": true, \"users\": { \"Bob\": { \"username\": \"${{ secrets.ZAC_TEST_USER_1_USERNAME }}\", \"password\": \"${{ secrets.ZAC_TEST_USER_1_PASSWORD }}\"}} }"'
```

Comment:
I would recommend to use e.g. `${{ secrets.ZAC_E2E_TEST_USER_1_USERNAME }}` here instead.